pages/api/user.ts

```diff
@@ -0,0 +1,29 @@
+import { withIronSessionApiRoute } from 'iron-session/next'
+import { sessionOptions } from 'lib/session'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+export type User = {
```

Comment:
För våran användare kanske vi bara sparar email och namn? 

Tänker att vi kan ha isLoggedIn om vi vill stödja visitors att ha kundvagnar, men då måste email och namn vara optional

Reply:
Added name instead of avatarUrl, but we'll discuss this further!